sdcm/cluster.py

```diff
@@ -880,6 +900,70 @@ def destroy(self):
         self.log.info('Destroyed')
 
 
+class GCENode(BaseNode):
+
+    """
+    Wraps GCE instances, so that we can also control the instance through SSH.
+    """
+
+    def __init__(self, gce_instance, gce_service, credentials,
+                 node_prefix='node', node_index=1, gce_image_username='root',
+                 base_logdir=None):
+        name = '%s-%s' % (node_prefix, node_index)
+        self._instance = gce_instance
+        self._gce_service = gce_service
+        self._wait_public_ip()
+        ssh_login_info = {'hostname': self.public_ip_address,
+                          'user': gce_image_username,
+                          'key_file': credentials.key_file,
+                          'extra_ssh_options': '-t'}
```

Comment:
Because sudo configuration in the CentOS images of these cloud providers require a tty, something that does not happen on AWS Scylla AMIs.